src/tests/adr/initializer_functions_adr.rs

```diff
@@ -161,7 +159,7 @@ fn initializers_are_assigned_or_delegated_to_respective_init_functions() {
     "###);
 
     // the init-function for `bar` will have a `CallStatement` to `__init_foo` as its only statement, passing the member-instance `self.fb`
-    let init_bar_impl = &units[1].implementations[2];
+    let init_bar_impl = &units[1].implementations[1];
```

Comment:
Certain initializers are now removed from the `HashMap` with `swap_remove`, so the ordering of some initializer functions has changed in older snapshots.